src/pybind/mgr/cephadm/templates/services/loki.yml.j2

```diff
@@ -0,0 +1,27 @@
+auth_enabled: false
+
+server:
+  http_listen_port: 3100
+  grpc_listen_port: 8080
+
+common:
+  path_prefix: /tmp/loki
+  storage:
+    filesystem:
+      chunks_directory: /tmp/loki/chunks
+      rules_directory: /tmp/loki/rules
+  replication_factor: 1
```

Comment:
This sounds interesting... I want to know more :D So it looks like Loki supports some kind of sharding/consistent hashing?

Reply:
Yes, Loki supports consistent hash rings - https://grafana.com/docs/loki/next/fundamentals/architecture/rings/